plugins/source/aws/resources/plugin/plugin_test.go

```diff
@@ -14,3 +14,31 @@ func TestAWS(t *testing.T) {
 		t.Errorf("Name() = %q, want %q", name, "aws")
 	}
 }
+
+// This test ensures that all tables have a unique description.
+func TestAWSTableDescriptions(t *testing.T) {
+	descriptions := make(map[string]string)
+	p := AWS()
+	tables := p.Tables()
+	for _, table := range tables {
```

Comment:
Need to update to handle nested tables
```suggestion
	for _, table := range tables.FlattenTables() {
```